runtime/src/vote_parser.rs

```diff
@@ -12,7 +12,7 @@ use {
 pub type ParsedVote = (Pubkey, VoteTransaction, Option<Hash>);
 
 // Used for filtering out votes from the transaction log collector
-pub(crate) fn is_simple_vote_transaction(transaction: &SanitizedTransaction) -> bool {
+pub fn is_simple_vote_transaction(transaction: &SanitizedTransaction) -> bool {
```

Comment:
Is this change still needed? I don't see your changeset calling this fn.

Reply:
No, I forget to remove it. Corrected.